code/modules/power/tesla/energy_ball.dm

```diff
@@ -40,17 +42,11 @@ var/list/blacklisted_tesla_types = list(/obj/machinery/atmospherics,
 /obj/singularity/energy_ball/process()
 	if(!is_orbiting)
 		handle_energy()
-		var/amount_to_move = 2 + orbiting_balls.len * 2
-		var/what_does_the_scouter_say_about_the_balls_power_level = (TESLA_DEFAULT_POWER + (TESLA_MINI_POWER * orbiting_balls.len))
-		move_the_basket_ball(amount_to_move)
-		pixel_x = 0
-		pixel_y = 0
+		move_the_basket_ball(2 + orbiting_balls.len * 2)
 		playsound(src.loc, 'sound/magic/lightningbolt.ogg', 100, 1, extrarange = 15)
-		tesla_zap(src, 7, what_does_the_scouter_say_about_the_balls_power_level)
-		pixel_x = -32
-		pixel_y = -32
-		energy += rand(1,3) // ensure it generates energy without needing to be blasted by the PA too much due to its size, and that a tesla engine will always get bigger over time
```

Comment:
The growth is so that it gets better and more dangerous over time, because people kept bitching "lol singulo gets more deadly and can escape over time, tesla is pussy engine that can't do that"